stock_move_location/wizard/stock_move_location.xml

```diff
@@ -64,11 +64,7 @@
                                     readonly="1"
                                     force_save="1"
                                 />
-                                <field
-                                    name="destination_location_id"
-                                    readonly="1"
-                                    force_save="1"
-                                />
+                                <field name="destination_location_id" force_save="1" />
```

Comment:
If you remove readonly, you can remove force_save

Reply:
Done!